addon/components/link-to.ts

```diff
@@ -309,7 +309,7 @@ const LinkComponent = EmberComponent.extend({
     @property disabledWhen
     @public
   */
-  disabledWhen: UNDEFINED,
+  disabledWhen: undefined,
```

Comment:
[this check](https://github.com/emberjs/ember-legacy-built-in-components/blob/main/addon/components/link-to.ts#L945) was making every link disabled hence clicks didn't result in transitions